packages/workbox-build/src/lib/use-build-type.js

```diff
@@ -0,0 +1,5 @@
+const DEFAULT_BUILD_TYPE = 'prod';
```

Comment:
This module doesn't necessarily have the most robust logic, and would need to change if we change our naming conventions.